src/components/App.js

```diff
@@ -8,4 +8,4 @@ class App extends React.Component {
   }
 }
 
-module.export = App;
\ No newline at end of file
+export default App; 
```

Comment:
And same thing here. Notice that I'm declaring `export default`. That removes the object wrapper from the component and export just there component itself.